app/src/main/res/layout/search_widget.xml

```diff
@@ -0,0 +1,34 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@drawable/search_widget_background"
```

Comment:
I spent a bit of time trying to make the widget have round corners, and ended up setting this custom background. It's not really a good way of doing this though, I was tempted to switch it back to as it was in #845, but thought I'd put it in this PR instead and see what people think. I haven't rounded the corners of the bookmarks widget, with this approach it felt like it'd probably be more complicated (and I think that widget looks ok as-is).